### PR TITLE
Add Foxy CI job for Performance Testing

### PR DESCRIPTION
**Pull request by ahcorde**
Add Foxy CI job for Performance Testing

Signed-off-by: ahcorde <ahcorde@gmail.com>

### CODE REVIEW

File: foxy/ci-benchmark.yaml [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+- ros-noetic-common-msgs
+- ros-noetic-rosbash
+- ros-noetic-roscpp
+- ros-noetic-roscpp-tutorials
+- ros-noetic-roslaunch
+- ros-noetic-rosmsg
+- ros-noetic-rospy-tutorials
+- ros-noetic-tf2-msgs
+jenkins_job_label: ci-agent
+jenkins_job_priority: 50
```
Comment (cottsay) [inline foxy/ci-benchmark.yaml]: This is the wrong job priority for Foxy. I believe 57 is the right priority.

**Review by cottsay — CHANGES_REQUESTED** [review]
This file needs to be referenced from the index to be picked up.

File: foxy/ci-benchmark.yaml (hunk 1 above)
Comment (dirk-thomas) [reply foxy/ci-benchmark.yaml]: Please post a list of all priorities to ease the review.

Comment (ahcorde) [reply foxy/ci-benchmark.yaml]: Updated: https://github.com/ros2/ros_buildfarm_config/pull/130/commits/955765e795a5c7db778c0e67f868315f0b251af8

By the way, Where I can check this information ?

Comment (dirk-thomas) [reply foxy/ci-benchmark.yaml]: > Where I can check this information ?

With the `list_priorities.py` script in the root of this repository.

Comment (ahcorde) [reply foxy/ci-benchmark.yaml]: yes, `foxy` jobs are 57:

```bash
57
  jenkins_commit_job_priority::foxy/source-build.yaml
  jenkins_job_priority::foxy/ci-nightly-connext.yaml
  jenkins_job_priority::foxy/ci-nightly-cross-vendor-connext-cyclonedds.yaml
  jenkins_job_priority::foxy/ci-nightly-cross-vendor-connext-fastrtps-dynamic.yaml
  jenkins_job_priority::foxy/ci-nightly-cross-vendor-connext-fastrtps.yaml
  jenkins_job_priority::foxy/ci-nightly-cross-vendor-cyclonedds-fastrtps-dynamic.yaml
  jenkins_job_priority::foxy/ci-nightly-cross-vendor-cyclonedds-fastrtps.yaml
  jenkins_job_priority::foxy/ci-nightly-cross-vendor-fastrtps-fastrtps-dynamic.yaml
  jenkins_job_priority::foxy/ci-nightly-cyclonedds.yaml
  jenkins_job_priority::foxy/ci-nightly-debug.yaml
  jenkins_job_priority::foxy/ci-nightly-extra-rmw-release.yaml
  jenkins_job_priority::foxy/ci-nightly-fastrtps-dynamic.yaml
  jenkins_job_priority::foxy/ci-nightly-fastrtps.yaml
  jenkins_job_priority::foxy/ci-nightly-performance.yaml
  jenkins_job_priority::foxy/ci-nightly-release.yaml
  jenkins_job_priority::foxy/ci-overlay.yaml
```

**Review by cottsay — APPROVED** [review]
Looks good. Let's get this deployed for tonight's build.

**Comment by cottsay** [comment]
> This file needs to be referenced from the index to be picked up.

Looks like this never happened, so the job isn't getting deployed.

**Comment by cottsay** [comment]
Here is the deployment job: [Build Status](http://build.ros2.org/job/Fci_reconfigure-jobs/296/)

**Pull request merged**